src/terminal_ui.cc

```diff
@@ -968,7 +968,7 @@ void TerminalUI::draw_menu()
 {
     // menu show may have not created the window if it did not fit.
     // so be tolerant.
-    if (not m_menu)
+    if (not m_menu or m_menu.size.line == 0)
```

Comment:
This does not look right to me, that condition would mean that m_menu.size.line is 0 but we still have allocated  the m_menu.lines arry, I think we should fix the logic that might lead to this inconsistency instead.

Reply:
I see, thank you for the feedback. After looking a bit, it seems that maybe the `m_menu.create()` should not create a menu if it won't fit in the window?
I will keep checking for a right solution :)